VP_DXVK_baseline.json

```diff
@@ -0,0 +1,374 @@
+                }
+            }
+        },
+        "vulkan11requirements": {
+            "extensions": {
+                "VK_KHR_16bit_storage": 1,
+                "VK_KHR_bind_memory2": 1,
+                "VK_KHR_dedicated_allocation": 1,
+                "VK_KHR_descriptor_update_template": 1,
+                "VK_KHR_device_group": 1,
+                "VK_KHR_device_group_creation": 1,
+                "VK_KHR_external_fence": 1,
+                "VK_KHR_external_fence_capabilities": 1,
+                "VK_KHR_external_memory": 1,
+                "VK_KHR_external_memory_capabilities": 1,
+                "VK_KHR_external_semaphore": 1,
+                "VK_KHR_external_semaphore_capabilities": 1,
+                "VK_KHR_get_memory_requirements2": 1,
+                "VK_KHR_get_physical_device_properties2": 1,
+                "VK_KHR_maintenance1": 1,
+                "VK_KHR_maintenance2": 1,
+                "VK_KHR_maintenance3": 1,
+                "VK_KHR_multiview": 1,
+                "VK_KHR_relaxed_block_layout": 1,
+                "VK_KHR_sampler_ycbcr_conversion": 1,
+                "VK_KHR_shader_draw_parameters": 1,
+                "VK_KHR_storage_buffer_storage_class": 1,
+                "VK_KHR_variable_pointers": 1
+            },
+            "features": {
+                "VkPhysicalDeviceMultiviewFeatures": {
+                    "multiview": true
+                }
+            },
+            "properties": {
+                "VkPhysicalDeviceMultiviewProperties": {
+                    "maxMultiviewViewCount": 6,
+                    "maxMultiviewInstanceIndex": 134217727
+                }
+            }
+        },
+        "vulkan12requirements": {
+            "extensions": {
+                "VK_KHR_8bit_storage": 1,
+                "VK_KHR_buffer_device_address": 1,
+                "VK_KHR_create_renderpass2": 1,
+                "VK_KHR_depth_stencil_resolve": 1,
+                "VK_KHR_draw_indirect_count": 1,
+                "VK_KHR_driver_properties": 1,
+                "VK_KHR_image_format_list": 1,
+                "VK_KHR_imageless_framebuffer": 1,
+                "VK_KHR_sampler_mirror_clamp_to_edge": 1,
+                "VK_KHR_separate_depth_stencil_layouts": 1,
+                "VK_KHR_shader_atomic_int64": 1,
+                "VK_KHR_shader_float16_int8": 1,
+                "VK_KHR_shader_float_controls": 1,
+                "VK_KHR_shader_subgroup_extended_types": 1,
+                "VK_KHR_spirv_1_4": 1,
+                "VK_KHR_timeline_semaphore": 1,
+                "VK_KHR_uniform_buffer_standard_layout": 1,
+                "VK_KHR_vulkan_memory_model": 1,
+                "VK_EXT_descriptor_indexing": 1,
+                "VK_EXT_host_query_reset": 1,
+                "VK_EXT_sampler_filter_minmax": 1,
+                "VK_EXT_scalar_block_layout": 1,
+                "VK_EXT_separate_stencil_usage": 1,
+                "VK_EXT_shader_viewport_index_layer": 1
```

Comment:
I don't understand these huge blocks of promoted extensions.

We only support Vulkan 1.3, and don't use any of them.

If this is a necessary quirk for profiles, then that should be fixed in the schema/spec/implementations/whatever.

Reply:
Ok removing the promoted extensions, there is no need for that.